client/lib/site-roles/test/store.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import useMockery from 'test/helpers/use-mockery';
+import useFakeDom from 'test/helpers/use-fake-dom';
+
+describe( 'Viewers Store', () => {
+	let Dispatcher, RolesStore;
+	let actions, site, siteId;
+
+	useMockery();
+	useFakeDom();
```

Comment:
yeah i've been dropping that in when I see `require( 'lib/react-test-env-setup' )();` in the test then circling back to see if its needed. In this case it isn't so I'll remove.